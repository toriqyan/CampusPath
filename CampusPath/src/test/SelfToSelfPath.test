# Verify that the program finds and correctly outputs the shortest path 
# from a place to itself
r
CSE
CSE

r
KNE
KNE

# All of test scripts would end with q to terminate the program.
q