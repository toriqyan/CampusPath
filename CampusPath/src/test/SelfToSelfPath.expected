Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify that the program finds and correctly outputs the shortest path 
# from a place to itself
Abbreviated name of starting building: Abbreviated name of ending building: Path from Paul G. Allen Center for Computer Science & Engineering to Paul G. Allen Center for Computer Science & Engineering:
Total distance: 0 feet

Enter an option ('m' to see the menu): 
Abbreviated name of starting building: Abbreviated name of ending building: Path from Kane Hall (North Entrance) to Kane Hall (North Entrance): 
Total distance: 0 feet

Enter an option ('m' to see the menu): 
# All of test scripts would end with q to terminate the program.